mne_qt_browser/_pg_figure.py

```diff
@@ -3690,7 +3689,21 @@ def closeEvent(self, event):
                 value = getattr(self.mne, qsetting)
                 QSettings().setValue(qsetting, value)
             self._close(event)
+
+        if hasattr(self, 'load_thread') and self.load_thread is not None:
```

Comment:
I prefer this more compact pattern but it doesn't really matter here
```suggestion
        if getattr(self, 'load_thread', None) is not None:
```